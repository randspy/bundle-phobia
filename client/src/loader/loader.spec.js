import React from 'react';
import { shallow } from 'enzyme';
import Loader from './loader';

describe('Loader', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
