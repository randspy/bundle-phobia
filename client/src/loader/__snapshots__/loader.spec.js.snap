// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<div
  className="loader"
>
  <Loader
    className=""
    color="black"
    height={40}
    style={Object {}}
    timeout={0}
    type="Grid"
    visible={true}
    width={40}
  />
</div>
`;
